### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by nuriddin2468**

### CODE REVIEW

File: src/app/modules/courses/components/course-card/course-card.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <div>{{course.creationDate | date}}</div>
+  </div>
+
+  <h2>{{course.title}}</h2>
+  <div class="description-wrapper">
+    <div class="description">
+      {{course.description}}
+    </div>
+    <div class="action">
+      <div><button (click)="edit.emit()">Edit</button></div>
```
Comment (goremchik) [inline src/app/modules/courses/components/course-card/course-card.component.html]: no need to wrap button in div, if you need each button on new line, just add proper styles

File: src/app/modules/courses/components/course-card/course-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.content {
+  margin-top: 20px;
+  display: flex;
+  padding: 30px 20px;
+  flex-direction: column;
+  background-color: white;
```
Comment (goremchik) [inline src/app/modules/courses/components/course-card/course-card.component.scss:6]: Create constants file and reuse it in all style files

File: src/app/modules/courses/types/course.ts [hunk 3]
```diff
@@ -2,6 +2,6 @@ export interface Course {
   id: number;
   title: string;
   creationDate: string;
-  duration: number;
+  duration: string;
```
Comment (goremchik) [inline src/app/modules/courses/types/course.ts]: actually duration should be a number, but in future HW (with custom pipe) you will show it as string, so no need to change data model

File: tsconfig.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "baseUrl": "src",
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
-    "strict": true,
+    "strict": false,
```
Comment (goremchik) [inline tsconfig.json:8]: Why do you change this property?

File: src/app/shared/components/wrapper/wrapper.component.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+:host {
+  height: 100vh;
```
Comment (goremchik) [inline src/app/shared/components/wrapper/wrapper.component.scss]: I think min-height, should be better here

File: src/app/shared/components/wrapper/header/header.component.html [hunk 6]
```diff
@@ -1 +1,10 @@
-<p>header works!</p>
+<div class="content">
+  <div>
+    <h2>Video Course</h2>
+  </div>
+
+  <div class="right-side">
```
Comment (goremchik) [inline src/app/shared/components/wrapper/header/header.component.html]: it is list, so use here: ul & li

**Review by goremchik — APPROVED** [review]

File: src/app/modules/courses/components/course-card/course-card.component.html (hunk 1 above)
Comment (nuriddin2468) [reply src/app/modules/courses/components/course-card/course-card.component.html]: fixed

File: src/app/modules/courses/types/course.ts (hunk 3 above)
Comment (nuriddin2468) [reply src/app/modules/courses/types/course.ts]: fixed

File: tsconfig.json (hunk 4 above)
Comment (nuriddin2468) [reply tsconfig.json:8]: i dont want to give initial value for my variables

File: src/app/shared/components/wrapper/wrapper.component.scss (hunk 5 above)
Comment (nuriddin2468) [reply src/app/shared/components/wrapper/wrapper.component.scss]: fixed

File: src/app/shared/components/wrapper/header/header.component.html (hunk 6 above)
Comment (nuriddin2468) [reply src/app/shared/components/wrapper/header/header.component.html]: fixed

**Review by goremchik — APPROVED** [review]

**Pull request merged**